cvm/docker-compose.yml

```diff
@@ -1,32 +1,55 @@
 services:
-  vllm:
+  vllm_service:
     image: vllm/vllm-openai:latest
-    container_name: vllm
-    environment:
-      - NVIDIA_VISIBLE_DEVICES=all
+    container_name: vllm_container
+    env_file:
+        - .env_prod
+    gpus: all
+    # runtime: nvidia
+    ports:
+      - "${PORT_PROVIDER}:${PORT_PROVIDER}"  # host:container port mapping
+    ipc: host
     volumes:
-      - huggingface-cache:/root/.cache/huggingface
-    runtime: nvidia
-    # No external ports - internal access only
-    expose:
-      - "8000"
-    networks:
-      - vllm
+      # - huggingface-cache:/root/.cache/huggingface
+      - ./tee:/tee
+      - /mnt/disk1/tee_models:/mnt/disk1/tee_models
+      - /mnt/disk1/huggingface:/root/.cache/huggingface
```

Comment:
I don't understand the volume renaming here. Why is the huggingface volume switched to a bind mount instead of named volume? What's the purpose of the two other volumes?